index.html

```diff
@@ -53,7 +55,7 @@ <h1 class="card-title">
                   id="city-input"
                   type="text"
                   class="form-control"
-                  placeholder="city"
+                  placeholder="city or city,country-code"
```

Comment:
I think just "city or country code".

Reply:
It is rome (city in the united states) or rome,it (city in Italy) , country code alone wouldn't be found.

Reply:
maybe I could write an example : city (ie. rome or rome, it)